tests/adapters/test_adapter.py

```diff
@@ -31,10 +31,17 @@
 
 @pytest.fixture()
 def adapter():
-    return Adapter(preprocess_reply=lambda v: v)
+    return Adapter()
 
 
-def test_init(adapter):
+@pytest.fixture()
+def fake():
+    return FakeAdapter()
+
+
+def test_init():
+    with pytest.warns(FutureWarning):
+        adapter = Adapter(preprocess_reply=lambda v: v)
```

Comment:
:heart: that you added a test for the deprecated feature! Maybe better as a separate test, though?
```
def test_deprecated_preprocess_reply():
    with pytest.warns(FutureWarning):
        adapter = Adapter(preprocess_reply=lambda v: v)
```

Reply:
The advantage of using `pytest.warns` is, that the warning is not printed to the test output. If I removed it, the test would output the warnings.

I can do an additional test as well.

Reply:
I see. Yeah, I thought you'd do the __init__ without the deprecated arg, then there won't be a warning.

Reply:
Of course. I change the test_init and added a test_deprecated...